apps/openmw/mwrender/sky.hpp

```diff
@@ -161,6 +163,8 @@ namespace MWRender
 
         void listAssetsToPreload(std::vector<std::string>& models, std::vector<std::string>& textures);
 
+        void setCamera(MWRender::Camera *camera);
```

Comment:
I was thinking of passing the osg::Camera actually. Smaller dependency, I guess.